README.md

```diff
@@ -385,6 +385,54 @@ command.
 Other options like `--wait-completion` are also available and work in the same way as for the `cloudos job run` command.
 Check `cloudos bash job --help` for more details.
 
+#### Send a bash array-job to CloudOS (parallel sample processing)
+
+The subcommand `bash array-job` can list the columns of an array file by using the option `--list-columns` when adding `--array-file <CloudOS File Explorer Path>`. The columns will be listed one per line, example:
+
+```console
+Columns:
+    - column1
+    - column2
+    - column3
+```
+
+When running a bash array job, the following options are available to customize the behavior:
+
+##### Array File
+- **`--array-file`**: Specifies the path to a file containing a set of columns useful in running the bash job. This option is **required** when using the command `bash array-job`.
+
+##### Separator
+- **`--separator`**: Defines the separator to use in the array file. Supported separators include:
+    - `,` (comma)
+    - `;` (semicolon)
+    - `tab`
+    - `space`
+    - `|` (pipe)
+This option is only applicable when `--array-file` is provided and is **required**.
+
+##### List Columns
+- **`--list-columns`**: Lists the columns available in the array file. This is useful for inspecting the structure of the file. This flag disables sending the job, it just prints the column list.
+
+##### Array File Project
+- **`--array-file-project`**: Specifies the name of the project to use when running the array file, if it is different from the project specified by `--project-name`.
+
+##### Disable Column Check
+- **`--disable-column-check`**: Disables the validation of columns in the array file. 
+
+##### Array Parameter
+- **`-a` / `--array-parameter`**: Allows passing individual parameters to the job call specifically for array parameters, specifying the column names present in the header. Each parameter should be in the format `parameter_name=parameter_value`. For example:
+    - `-a --test=value` or
+    - `--array-parameter -test=value`
+specify a column named 'value' in the array file header. Adding array parameters not present in the header will cause an error. This option can be used multiple times to include as many array parameters as needed.
```

Comment:
Sorry, I don't understand: is the `value` what should correspond with a column in array file, right (and that's normally checked and not checked when `--disable-column-check`). And parameter should correspond to a variable in your command, correct? That should be explained in a better way in my opinion